packages/devextreme-react/src/core/template-wrapper.ts

```diff
@@ -46,10 +47,12 @@ class TemplateWrapper extends React.PureComponent<ITemplateWrapperProps, ITempla
   public componentWillUnmount(): void {
     // Let React remove it itself
     const node = this.element;
+    const hiddenNode = this.hiddenElement;
     const { container } = this.props;
 
-    if (node) {
+    if (node && hiddenNode) {
```

Comment:
It seems that we can caught a case, when `hiddenNode` exists, but `node` doesn't.
Should we separate this by few `if`s?